File: Aula2/DivResto/DivResto.cpp
#include <iostream>

int main()
{
    int a, b, div, resto;
    
    std::cin >> a >> b;
    
    div = a / b;
    resto = a % b;
    
    std::cout << div << " " << resto << std::endl;    
    
    return 0;
}

File: Aula1/DivResto/DivResto.java
import java.util.Scanner;

public class DivResto{
    public static void main (String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int a = sc.nextInt();
        int b = sc.nextInt();
        int div = a/b;
        int resto = a%b;
        
        System.out.println(div + " " + resto);
    }
}